apps/chat_websocket/config/prod.exs

```diff
@@ -14,9 +14,12 @@ use Mix.Config
 # manifest is generated by the mix phx.digest task
 # which you typically run after static files are built.
 config :chat_websocket, ChatWebsocket.Endpoint,
-  load_from_system_env: true,
-  url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  load_from_system_env: false,
+  http: [port: System.get_env("CHAT_WEBSOCKET_PORT")],
+  url: [host: "127.0.0.1", port: 80],
+  check_origin: false,
```

Comment:
As I understand it the [check_origin](https://github.com/phoenixframework/phoenix/blob/master/lib/phoenix/transports/websocket.ex#L21) validates the headers from requests and would want to be on in a true "prod" environment? 

At this time this is set to false for easier testing?